examples/statistics/violinplot.py

```diff
@@ -28,59 +28,60 @@
 pos = [1, 2, 4, 5, 7, 8]
 data = [np.random.normal(0, std, size=100) for std in pos]
 
-fig, axes = plt.subplots(nrows=2, ncols=5, figsize=(10, 6))
+fig, axs = plt.subplots(nrows=2, ncols=5, figsize=(10, 6))
 
-axes[0, 0].violinplot(data, pos, points=20, widths=0.3,
-                      showmeans=True, showextrema=True, showmedians=True)
-axes[0, 0].set_title('Custom violinplot 1', fontsize=fs)
+axs[0, 0].violinplot(data, pos, points=20, widths=0.3,
+                     showmeans=True, showextrema=True, showmedians=True)
+axs[0, 0].set_title('Custom violinplot 1', fontsize=fs)
 
-axes[0, 1].violinplot(data, pos, points=40, widths=0.5,
-                      showmeans=True, showextrema=True, showmedians=True,
-                      bw_method='silverman')
-axes[0, 1].set_title('Custom violinplot 2', fontsize=fs)
+axs[0, 1].violinplot(data, pos, points=40, widths=0.5,
+                     showmeans=True, showextrema=True, showmedians=True,
+                     bw_method='silverman')
+axs[0, 1].set_title('Custom violinplot 2', fontsize=fs)
 
-axes[0, 2].violinplot(data, pos, points=60, widths=0.7, showmeans=True,
-                      showextrema=True, showmedians=True, bw_method=0.5)
-axes[0, 2].set_title('Custom violinplot 3', fontsize=fs)
+axs[0, 2].violinplot(data, pos, points=60, widths=0.7, showmeans=True,
+                     showextrema=True, showmedians=True, bw_method=0.5)
+axs[0, 2].set_title('Custom violinplot 3', fontsize=fs)
 
-axes[0, 3].violinplot(data, pos, points=60, widths=0.7, showmeans=True,
+axs[0, 3].violinplot(data, pos, points=60, widths=0.7, showmeans=True,
```

Comment:
alignment (a bunch of them below, too)

Reply:
Ha, got broken again due to rebasing :sad:. Thanks, fixed.